overlay.d/15fcos/usr/lib/systemd/system/coreos-migrate-systemd-resolved.service

```
@@ -0,0 +1,20 @@
+# This unit is used to migrate existing systems running default
+# configurations to systemd-resolved as part of the Fedora 33 change.
+# https://fedoraproject.org/wiki/Changes/systemd-resolved
+[Unit]
+Description=CoreOS Migrate to Systemd Resolved
+Documentation=https://github.com/coreos/fedora-coreos-tracker/issues/646
+# Run before NetworkManager and systemd-resolved start
+Before=NetworkManager.service
+Before=systemd-resolved.service
+# Ensure we only run once on a F33 system. This file is created
+# by the coreos-migrate-systemd-resolved script
+ConditionPathExists=!/var/lib/coreos-migrate-systemd-resolved-ran
+
+[Service]
+Type=simple
```

```suggestion
Type=oneshot
```

I don't think we want this to act like a daemon.

---

Yeah I copied the unit file from another unit (`coreos-liveiso-success.service`) and forgot to update it.